project/ISSUES.md

```diff
@@ -0,0 +1,179 @@
+justifications* (e.g. the reason why it's a wontfix, or a reference to the
+duplicate issue) should be indicated in the closing comment by the issue
+closer.
+
+
+Area Labels
+------------
+
+The 'area' label is used for tagging issues and pull requests to a particular
+focus area.  This allows for easy searching within that particular domain, as
+well as more organized release notes.  Some examples, not-exhaustive, are here:
+
+* `area/API`
+* `area/cleanup`
+* `area/jobs`
+* `area/tests`
+* `area/GIEs`
+* `area/toolshed`
+* `area/UI-UX`
+* `area/workflows`
+
+This list will definitely grow over time.
+
+Other Useful Labels
+-------------------
+
+While the three labels sets indicating kind, status, and area are required
+there are several other labels that are be useful and/or have special purpose.
+
+* `procedures` is a special tag that indicates that the issue is related to
+  project governance, and it overrides the need for the trio of
+  kind/status/area tags, and these are never auto-flagged for triage.
+
+* `planning` is also a special tag that indicates the issue is related to
+  larger-scale issue planning.  These issues are typically meta-issues
+  containing checklists and references to other issues which are subcomponents
+  and stepping stones necessary for issue resolution.  These *can* utilize the
+  `focus/*` tags but are not required to.  Status and type make little sense
+  here.
+
+* `roadmap` is a reserved tag for the primary project roadmap.  This is a
+  meta-issue that is not expected to be completed, but rather serves as an
+  entry point to the high level development of the project.
+
+* `beginner-friendly` can be used to indicate a nice entry-level issue that
+  only requires limited understanding of the larger Galaxy framework and
+  ecosystem.  This is useful for encouraging new contributors.
+
+
+The Roadmap
+===========
+
+We will maintain a single `Roadmap` tagged meta-issue which will describe (at a
+very high level) the *current* major areas of focus for the project.  This is
+similar to our PRIORITIES 2014/15 cards on Trello.  Using [Task
+Lists](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments),
+this issue will link to sub-issues which will go into much more detail, might
+have its own checklists to even more subcomponent cards, and so on.  
+
+This `Roadmap` issue will be assigned to every release milestone, forcing
+periodic review of the roadmap.
+
+To prevent the roadmap from being tied completely to GitHub, and to facilitate
+portable change tracking over time, we will also maintain the file
+project/ROADMAP.md within the repository.  Whenever the ROADMAP issue text is
+changed, ROADMAP.md should be updated correspondingly.
+
+This document won't have the organizational integration that a live GitHub
+issue does, but this way we're be able to have a ROADMAP.md permanently
+attached to the code regardless of what issue tracking or organizational
+software we use in the future.
```

Comment:
> I feel like the problem with redoing the roadmap every few months is that we never seem to cut it down. I feel like every time we tried to adjust the one on Trello we would just add new stuff to it (planemo, IEs, etc...) and it never got smaller.

This is the motivation beyond redoing the roadmap and not updating it. There has to be an accompanying social piece here in which we actually consider each item on the roadmap and figure our whether it is still alive and where it should go. Roadmap is a bit more than just priorities, we should also put timelines on things (even if the granualarity, is "next release", "this year", "soon", and "the distant future"). Will this work? I don't know. Is it worth trying? I think so.

> individual focused one and not an organizational focused one 

Is this a feature or a bug?

> So what if we leverage the fact we are different and use it to ensure the roadmap stays trim. We have a roadmap issue and every committer gets one line to discuss what they intend to work on for the next three months.

I think what you are proposing just reinforces what we currently have. I think the goal here is to push for a little bit of change. We routinely hear complaints that we lack a transparent high-level roadmap, and that people outside (and sometimes inside) the team aren't clear on the big picture priorities. 

> Anton is going to have a tremendous amount of say in what I and Martin work on for instance, and James will have a tremendous amount of say in what Dannon and Carl work on but ultimately that is reflected through the individual 

Is this really where we want to be? Even now, @nekrut and @jxtx make prioritization decisions together, with input from everyone on their teams. Here we have the potential to open this up, and make it more transparent and inclusive. 

Sure, we can just have a list of what each person thinks they are working on, but that loses all of the planning behind things. And it is inherently isolating.